aea/components/loader.py

```diff
@@ -54,18 +54,15 @@ def component_type_to_class(component_type: ComponentType) -> Type[Component]:
 
 
 def load_component_from_config(  # type: ignore
```

Comment:
why does this have a type ignore?

Reply:
```aea/components/loader.py:56: error: Missing return statement```

this is because we ignore the `return` statement due to the `from_config` method.